sql/src/main/java/org/apache/druid/sql/calcite/schema/SystemSchema.java

```diff
@@ -1042,7 +1035,7 @@ private static JsonParserIterator<SupervisorStatus> getSupervisors(
       ObjectMapper jsonMapper
   )
   {
-    return getThingsFromLeaderNode(
+    return MetadataSegmentView.getThingsFromLeaderNode(
```

Comment:
It doesn't make sense to have `SystemSchema` call a static method on `MetadataSegmentView`. If you want to reuse this method, maybe consider putting it in some other existing utility class. You could probably even have it as a non-static method on the `DruidLeaderClient` itself.